scripts/rename_session_windows.py

```diff
@@ -99,14 +126,24 @@ def get_panes_programs(session: libtmux.Session, options: Options):
     return [Pane(p, get_current_program(running_programs, int(p['pane_pid']), options.shells)) for p in session_active_panes]
 
 def rename_windows(server: libtmux.Server):
+    is_running = get_option(server, 'running', 0)
+    if is_running:
+        return
+    set_option(server, 'running', '1')
```

Comment:
I think a context manager will be great for the `running` option and the disable/enable of `user_rename_hook`